curriculum/challenges/english/25-front-end-development/quiz-css-animations/66ed8fc9f45ce3ece4053eae.md

```diff
@@ -17,439 +17,419 @@ Answer all of the questions below correctly to pass the quiz.
+Fades in and out during each step.
 
 #### --answer--
 
-Placeholder answer
+Jumps immediately to the next frame.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of focus indicators?
 
 #### --distractors--
-
-Placeholder distractor 1
-
+To show hidden elements.
 ---
-
-Placeholder distractor 2
-
+To enhance font styles.
 ---
-
-Placeholder distractor 3
-
+To adjust the layout.
 #### --answer--
-
-Placeholder answer
-
+To highlight focused elements
 ### --question--
 
 #### --text--
-
-Placeholder question
+What will happen when this animation `transform: translateX(200px)` is applied?
 
 #### --distractors--
 
-Placeholder distractor 1
+The element will move left.
 
 ---
 
-Placeholder distractor 2
+`The element will move 200px to the bottom.
 
 ---
 
-Placeholder distractor 3
+The element will rotate 200 degrees.
 
 #### --answer--
 
-Placeholder answer
+The element will move 200px to the right.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How will the animation behave if `animation-iteration-count` is set to `infinite`?
 
 #### --distractors--
 
-Placeholder distractor 1
+The animation will run once and stop.
 
 ---
 
-Placeholder distractor 2
+The animation will pause after the first iteration.
 
 ---
 
-Placeholder distractor 3
+The animation will stop after three iterations.
 
 #### --answer--
 
-Placeholder answer
+The animation will repeat indefinitely.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which `keyframe` selector defines the halfway point of an animation?
 
 #### --distractors--
 
-Placeholder distractor 1
+`0%`
 
 ---
 
-Placeholder distractor 2
+`25%`
 
 ---
 
-Placeholder distractor 3
+`100%`
 
 #### --answer--
 
-Placeholder answer
+`50%`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the animation shorthand property include?
 
 #### --distractors--
 
-Placeholder distractor 1
+Only animation name
 
 ---
 
-Placeholder distractor 2
+Only animation timing
 
 ---
 
-Placeholder distractor 3
+Only animation duration
 
 #### --answer--
 
-Placeholder answer
+All animation properties
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the effect of using the `ease-in-out` timing function?
 
 #### --distractors--
 
-Placeholder distractor 1
+The animation will start fast and end slow
 
 ---
 
-Placeholder distractor 2
+The animation will run at a constant speed.
 
 ---
 
-Placeholder distractor 3
+The animation will loop infinitely.
 
 #### --answer--
 
-Placeholder answer
+The animation will start slow, speed up, then slow down.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property prevents the animation from running immediately?
 
 #### --distractors--
 
-Placeholder distractor 1
+`animation-fill-mode`
 
 ---
 
-Placeholder distractor 2
+`animation-timing-function`
 
 ---
 
-Placeholder distractor 3
+`animation-iteration-count`
 
 #### --answer--
 
-Placeholder answer
-
+`animation-delay`
```

Comment:
```suggestion
`animation-delay`

```

Reply:
this last one needs to be addressed still